src/util/Timer.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Are we planning on adding to this? We don't save any code by using this over performance.mark directly with the current implementation.